### PR TITLE
Messages websockets

### PR DESCRIPTION
**Pull request by nuzurie**
Connections:
* Receive messages (both new and get updates on screen when others edit and delete messages)
* Edit your own messages 
* Delete your own messages

### CODE REVIEW

File: src/screens/Messages/Tabs/TeamChat.js [hunk 1]
```diff
@@ -69,16 +66,13 @@ class TeamChat extends Component {
     navigateToChat(mockConversation, room) {
         this.props.navigation.navigate('ConversationScreen', {
             conversation: mockConversation,
-            room: room,
-            userID: this.state.userID,
-            getChatRooms: this.getChatRooms
```
Comment (car12lin12) [inline src/screens/Messages/Tabs/TeamChat.js]: please add [getChatRooms ](https://github.com/Airbenders-490/AirbendersFE/blob/bdaaa94673ad64e7d8758a74479a50e394b9757e/src/screens/Messages/Tabs/TeamChat.js#L74) back, it is needed for accept join request

File: src/screens/Classes.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { StyleSheet, Text, View, Image } from 'react-native';
+import theme from '../styles/theme.style.js';
+
+import ScreenContainer from '../containers/ScreenContainer';
+import { Title, Subtitle, TextBody } from '../containers/TextContainer.js';
+import MainContainer from '../containers/MainContainer.js';
+import AddTeamButton from '../components/AddButton.js';
+import MessageInput from '../components/MessageInput.js';
+
+class Classes extends Component {
```
Comment (car12lin12) [inline src/screens/Classes.js]: where do we use this component?

File: src/screens/Messages/MessageTabs.js [hunk 3]
```diff
@@ -14,34 +14,36 @@ class MessageTabs extends Component {
...
-          screenOptions={{
-            tabBarLabelStyle: {
-              fontSize: theme.FONT_SIZE_SLIGHT_MEDIUM,
-              fontFamily: theme.FONT_SEMIBOLD,
-              letterSpacing: theme.LETTER_SPACING_SMALL,
-            },
-            tabBarStyle: {
-              backgroundColor: "transparent",
-              marginBottom: theme.SPACING_SMALL,
-              marginTop: theme.SPACING_LARGE, /* temporary marginTop */
```
Comment (car12lin12) [inline src/screens/Messages/MessageTabs.js:28]: can you add marginTop back? it is very far to the top on my smaller android screen

File: src/screens/Messages/Tabs/TeamChat.js [hunk 4]
```diff
@@ -69,16 +66,13 @@ class TeamChat extends Component {
...
+            room: room
         })
     }
 
     removeTeam = async (room) => {
 
         let token = await AsyncStorage.getItem("token")
-        let user = await AsyncStorage.getItem("userID")
```
Comment (car12lin12) [inline src/screens/Messages/Tabs/TeamChat.js]: can you add [userID ](https://github.com/Airbenders-490/AirbendersFE/blob/bdaaa94673ad64e7d8758a74479a50e394b9757e/src/screens/Messages/Tabs/TeamChat.js#L81) back please

**Review by car12lin12 — CHANGES_REQUESTED** [review]
Great job merging guys! A few changes are needed for unexpected behaviors

File: src/screens/Messages/Tabs/TeamChat.js (hunk 1 above)
Comment (nuzurie) [reply src/screens/Messages/Tabs/TeamChat.js]: oops

File: src/screens/Classes.js (hunk 2 above)
Comment (yasfahm) [reply src/screens/Classes.js]: this component can be removed

**Review by car12lin12 — APPROVED** [review]
:rocket::rocket:

File: src/screens/Classes.js (hunk 2 above)
Comment (nuzurie) [reply src/screens/Classes.js]: Idk where it was added (there are like 100 commits here). But @stellakimnguyen wanted to work on the app after the deadline and might want to use this.

Comment (yasfahm) [reply src/screens/Classes.js]: You can remove this entirely because we replaced it with Teams.js

**Review by yasfahm — APPROVED** [review]

**Review by car12lin12 — APPROVED** [review]

**Pull request merged**